specs/protocol/jovian/exec-engine.md

```diff
@@ -0,0 +1,70 @@
+# Jovian: Execution Engine
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Minimum Base Fee](#minimum-base-fee)
+  - [Minimum Base Fee in Block Header](#minimum-base-fee-in-block-header)
+  - [Minimum Base Fee in `PayloadAttributesV3`](#minimum-base-fee-in-payloadattributesv3)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Minimum Base Fee
+
+Jovian introduces a configurable minimum base fee to reduce the duration of priority-fee auctions on OP Stack chains.
```

Comment:
You could link out to the design doc here in case the user wants to know more about the motivation.